extensions/jfr/runtime/src/main/java/io/quarkus/jfr/runtime/SpanIdRelational.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.jfr.runtime;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import jdk.jfr.*;
```

Comment:
Please avoid star imports

Reply:
I have reviewed imports in all classes and changed to single class imports